tika-core/src/main/java/org/apache/tika/sax/XHTMLContentHandler.java

```diff
@@ -138,7 +138,12 @@ private void lazyStartHead() throws SAXException {
             
             // Call directly, so we don't go through our startElement(), which will
             // ignore these elements.
-            super.startElement(XHTML, "html", "html", EMPTY_ATTRIBUTES);
+            AttributesImpl htmlAttrs = new AttributesImpl();
```

Comment:
Thank you for the patch.  This feels fiddly to me.  Let's leave XHTMLContentHandler alone.

Reply:
But how can we fix [TIKA-2100](https://issues.apache.org/jira/browse/TIKA-2100) without touching XHTMLContentHandler ?

Reply:
Sorry...I'm so narrowly focused only on my own typical use case - extract text without structure, put the lang in Metadata()...

To confirm, you have a need to include the 'lang=' in the xhtml output from Tika?  Are there other common attributes of HTML that we should also include?

Reply:
Hmm, did you mean attributes of the `<html>` tag, or html attributes in general ?

Reply:
Sorry, right, attributes of `<html>`.

Reply:
let's include :lang in the HTML attributes as well +1.

Reply:
So the answer is no, I do not need other attributes of the `<html>` tag.